bot/__init__.py

```diff
@@ -16,16 +15,7 @@
 if os.name == "nt":
     asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
 
-monkey_patches.patch_typing()
-
-# This patches any convertors that use PartialMessage, but not the PartialMessageConverter itself
-# as library objects are made by this mapping.
-# https://github.com/Rapptz/discord.py/blob/1a4e73d59932cdbe7bf2c281f25e32529fc7ae1f/discord/ext/commands/converter.py#L984-L1004
-commands.converter.PartialMessageConverter = monkey_patches.FixedPartialMessageConverter
-
-# Monkey-patch discord.py decorators to use the Command subclass which supports root aliases.
-# Must be patched before any cogs are added.
-commands.command = partial(commands.command, cls=monkey_patches.Command)
-commands.GroupMixin.command = partialmethod(commands.GroupMixin.command, cls=monkey_patches.Command)
+# Apply all monkey patches from bot core.
+monkey_patches.apply_monkey_patches()
```

Comment:
This API is kind of silly. If there's only going to be one public function, it would have been better to expose it directly from the utils namespace. You stated yourself that we'd probably never want to apply patches on an individual basis; if patches are no longer needed we'd just remove them entirely from the code base.

Reply:
@MarkKoz I think you should open an issue in the bot-core repository then. I agree with you and would like to see this changed, but at the moment this is not something Chris can change in this PR other than doing `from botcore.utils.monkey_patches import apply_monkey_patches`.

Reply:
Raised https://github.com/python-discord/bot-core/pull/35 for this comment, good point Mark

Reply:
Fixed in https://github.com/python-discord/bot/pull/2103/commits/9e6989aee80ee21a2f3defa94d9eca36834fa771